src/server/reset_to_version.rs

```diff
@@ -215,8 +216,16 @@ impl ResetToVersionManager {
         let readopts = IterOptions::new(None, None, false);
         let write_iter = self
             .engine
-            .iterator_opt(CF_WRITE, readopts.clone())
+            .iterator_opt(CF_WRITE, {
+                // Filter out the SST that does not have a newer version than `ts` in
+                // `CF_WRITE`.
+                let mut cf_write_readopts = readopts.clone();
+                cf_write_readopts.set_hint_min_ts(Bound::Excluded(ts.into_inner()));
```

Comment:
seems should use `hint_max_ts`?

Reply:
What we need to do here is to filter out the SST whose max TS is smaller than the TS we want to reset, in this case, we don't need to delete any key from it.

Reply:
oh, my bad